src/extension/background-script/actions/allowances/__tests__/enable.test.ts

```diff
@@ -59,7 +59,7 @@ describe("enable allowance", () => {
     const message: MessageAllowanceEnable = {
       application: "LBE",
       prompt: true,
-      action: "enableAllowance",
+      action: "public/webln/enable",
```

Comment:
I think this is never `public/...` is it?

Reply:
That's what it is currently?:

https://github.com/getAlby/lightning-browser-extension/blob/master/src/app/router/Prompt/Prompt.tsx#L68

![image](https://user-images.githubusercontent.com/1016218/204559236-ed635d3b-6f69-4f21-a93f-7b16bcc26b53.png)

Reply:
true, sorry. that was added with nostr.